README.md

```diff
@@ -1,2 +1,38 @@
 # CityApp-ANET
+
 A simple web application that displays a list of cities
+
+## Prerequisites
+
+* [Docker](https://www.docker.com/)
+* [Docker Compose](https://docs.docker.com/compose/)
+* Run all containers with `docker compose up`
+
+1. Database will run on localhost:5432. 
+1. Frontend will run on http://localhost:5083/api.
+1. Backend  will run on http://localhost:8080
+
+
+## Tech stack
+
+* Database: [PostgreSQL](https://www.postgresql.org/)
+* Backend:  [.NET](https://dotnet.microsoft.com)
+* Frontend: [Angular](https://github.com/angular)
+
+
+## Teamwork notes
+
+* Team setup: Marko (BE, FE), Kristjan (FE), Merily (DevOps, Design),
+* Worked hours: 
+  
+* Benefits of working in a team:
+1. More efficient planning for all parts
+1. Dividing tasks means better focus, faster development and richer implementation
+1. `````fill here smth`````
+
+**Note: Initially we divided work of each area to separate person. 
+When one of the developers fell ill, others took over to 
+finish the work while following the design agreed beforehand.**
+```Add takeover process```
```

Comment:
Takeover process was fairly simple:
* All made changes were pushed into a separate branch
** The developer who fell ill also included comments and todos in code and README to ease the takeover for other developers on the team) 
* Information related to the branch was shared with teammates (or a teammate who was going to take over the task)